Assets/PatchKit Patcher/Scripts/AppData/Local/Pack1Meta.cs

```diff
@@ -34,9 +34,14 @@ public static Pack1Meta ParseFromFile(string filename)
         public static Pack1Meta Parse(string content)
         {
             DebugLogger.Log(content);
-            return JsonConvert.DeserializeObject<Pack1Meta>(content);
-        }
+            Pack1Meta pack = JsonConvert.DeserializeObject<Pack1Meta>(content);
+            foreach (var fileEntry in pack.Files)
```

Comment:
Replace `var` with a specific type name if it cannot be determined by looking at this line.

Reply:
Outdated, I changed this for something else